datasets/recipes/minimal_recommended.json

```diff
@@ -0,0 +1,49 @@
+{
+  "@context": {
+    "@vocab": "https://schema.org/",
+    "sc": "https://schema.org/",
+    "ml": "http://mlcommons.org/schema/",
+    "recordSet": "ml:RecordSet",
```

Comment:
Down the line, we probably want to shove all these definitions into an external context file, as specified in:
https://www.w3.org/TR/json-ld/#advanced-context-usage

I'm not 100% sure this is supported by Search engines though.

Reply:
https://github.com/mlcommons/datasets_format/issues/42 to track.